sources/lib/Session/Session.php

```diff
@@ -291,6 +291,45 @@ public function getClientUsingPooler($type, $identifier)
     }
 
     /**
+     * Get query manager shortcut
+     *
+     * @param   string                   $identifier
+     * @throws  FoundationException      if no poolers found
+     * @return  \PommProject\Foundation\QueryManager\QueryManagerClient
+     */
+    public function getQueryManager($identifier = null)
+    {
+        return $this->getClientUsingPooler('query_manager', $identifier);
+    }
+
+    /**
+     * Get prepared query manager shortcut
+     *
+     * @param   string $identifier
+     * @throws  FoundationException      if no poolers found
+     * @return  \PommProject\Foundation\PreparedQuery\PreparedQueryManager
+     */
+    public function getPreparedQueryManager($identifier = null)
+    {
+        return $this->getClientUsingPooler('prepared_query', $identifier);
+    }
+
+    /**
+     * Get model shortcut
+     *
+     * Without the pomm-project/model-manage package or if the class name
+     * does not exist, this method raise a FoundationException.
+     *
+     * @param   string                   $className model class name
+     * @throws  FoundationException      if no poolers found
+     * @return  \PommProject\ModelManager\Model\Model
```

Comment:
I am referencing here a non existing class, that is from the model-manager package. This doesn't hurt since this is only in the PHPdoc, because PHP cannot properly type hint return types; but this might cause a dependency problem somehow.

Maybe a cleaner to extend the session class into the Symfony bundle and expose this method there.

What's your opinion about this?